website/common/locales/en/spells.json

```diff
@@ -77,5 +77,6 @@
   "challengeTasksNoCast": "Casting a skill on challenge tasks is not allowed.",
   "groupTasksNoCast": "Casting a skill on group tasks is not allowed.",
   "spellNotOwned": "You don't own this skill.",
-  "spellLevelTooHigh": "You must be level <%= level %> to use this skill."
+  "spellLevelTooHigh": "You must be level <%= level %> to use this skill.",
+  "sentCardToUser": "You sent a card to <%= userName %>"
```

Comment:
Could the new string be moved to the `generic.json` file, just under the `Cards` line (about liine 175)? That would keep it with similar strings which would help us if we were modifying them all in some future change.

Reply:
will do! I always try to find the right spot, but sometimes it's possible haha, good catch :)